maven/hw05-Annotations/src/main/java/ru/otus/Annotations/Annotations.java

```diff
@@ -0,0 +1,12 @@
+package ru.otus.Annotations;
+
+
+public class Annotations {
+    public static void main(String[] args) throws Exception {
+       for (int i=0; i<100; i++) {
+           TestLaunch.launch("ContainerTest");
```

Comment:
Лучше передать полное имя класса
```suggestion
           TestLaunch.launch("ru.otus.Annotations.ContainerTest");
```

Чтобы не было привязки к пакету внутри `TestLaunch` и мы могли передать в `TestLaunch` любой класс из любого пакета.